cmd/root.go

```diff
@@ -112,7 +124,14 @@ There will be a ClusterPolicyReport with results for cluster-wide resources.`,
 			if err != nil {
 				return err
 			}
-			policyReportStore := report.NewPolicyReportStore(client)
+			var policyReportStore report.ReportStore
+			if len(suseObsURL) > 0 && len(suseObsApiKey) > 0 && len(suseObsUrn) > 0 && len(suseObsCluster) > 0 {
+				log.Debug().Msg("Using SUSE Observability as report store")
+				policyReportStore = report.NewSuseObsStore(suseObsApiKey, suseObsURL, suseObsUrn, suseObsCluster, suseObsRepeatInterval, suseObsExpireInterval)
+			} else {
+				log.Debug().Msg("Using Kubernetes as report store")
+				policyReportStore = report.NewPolicyReportStore(client)
+			}
```

Comment:
Yes, this is very ugly. And it needs to be refactored. 

What do you think about having multiple stores? For example, the audit scanner could create policy reports and send the data to SUSE Obs.